lib/mrb/scripts/logger/flags.rb

```diff
@@ -0,0 +1,36 @@
+module Groonga
+  class Logger
+    class Flags
```

Comment:
Why `Flags` not `Flag`?

Reply:
Oh, for consistency, it seems that `Flag` is better.

Reply:
I've fixed.